en/controllers/components/authentication.rst

```diff
@@ -455,9 +455,9 @@ other password hashes, based on the RFC for digest authentication.
 
 .. note::
 
-    The third parameter of DigestAuthenticate::password() must match the
+    The second parameter of DigestAuthenticate::password() must match the
```

Comment:
@ynaderi still seeing this.

Reply:
I don't know how make undo a commit.
Please do it yourself.
tx

Reply:
![image](https://user-images.githubusercontent.com/30121709/32391471-8f5ed4cc-c0a8-11e7-8add-b6ad1c52916a.png)

I don't have access to change the master document.

Reply:
You have to push to your own branch ``ynaderi:3.0``. You either cloned the wrong repo (the main repo) or misconfigured your clone's remotes wrong.

You should have a clone of ``https://github.com/ynaderi/docs.git``.

Reply:
Thanks.
it works